### PR TITLE
creating no_sir.py: to tell users to not use 'sir'

### PR DESCRIPTION
**Pull request by shaipranesh11**
Ask people to not use 'sir' when they do #337.

# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: plugins/no_sir.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from errbot import BotPlugin
+
+
+class no_sir(BotPlugin):
+    """do not use sir to sound more friendly"""  # Ignore QuotesBear
+
+    def callback_message(self, msg):
```
Comment (LifetimeAlone) [inline plugins/no_sir.py]: It's better to use `@re_botcmd`, check other modules for some useful patterns :wink:

File: plugins/no_sir.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        emots = [':D']
+        if match_sir:
+            self.send(
+                msg.frm,
+                '@{}, donot use \'sir\' in ur conversation. {}'.format(
```
Comment (LifetimeAlone) [inline plugins/no_sir.py]: This indentation looks quite weird and it's hard to read it, try to make it a bit more compact :)

File: plugins/no_sir.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+from errbot import BotPlugin
```
Comment (LifetimeAlone) [inline plugins/no_sir.py]: Please, add tests and no_sir.plug. 
See other .plug files in plugins/ directory and tests in tests/ directory.
New modules can't be added without .plug and testing :cry:

**Comment by shaipranesh11** [comment]
what's wrong with this?

File: plugins/no_sir.py (hunk 2 above)
Comment (shaipranesh11) [reply plugins/no_sir.py]: k

File: plugins/no_sir.py (hunk 3 above)
Comment (shaipranesh11) [reply plugins/no_sir.py]: k thanks

**Pull request closed without merging**